lib/cc/analyzer/container.rb

```diff
@@ -178,9 +176,11 @@ def kill_wait_thread
 
       def reap_running_container(message)
         Analyzer.logger.warn("killing container name=#{@name} message=#{message.inspect}")
-        POSIX::Spawn::Child.new("docker", "kill", @name, timeout: 2.minutes)
-        POSIX::Spawn::Child.new("docker", "wait", @name, timeout: 2.minutes)
-      rescue POSIX::Spawn::TimeoutExceeded
+        Timeout.timeout(2.minutes.to_i) do
+          Kernel.system("docker kill #{@name}")
+          Kernel.system("docker wait #{@name}")
```

Comment:
```suggestion
          system("docker", "kill", @name)
          system("docker", "wait", @name)
```
This way you avoid the shell which is generally preferred. See https://apidock.com/ruby/Kernel/system